src/wallet/userPem.ts

```diff
@@ -0,0 +1,59 @@
+import { PathLike, readFileSync, writeFileSync } from "fs";
+import { isAbsolute, join, resolve } from "path";
+import { PemEntry } from "./pemEntry";
+import { USER_SEED_LENGTH, UserPublicKey, UserSecretKey } from "./userKeys";
+
+export class UserPem {
+    label: string;
+    secretKey: UserSecretKey;
+    publicKey: UserPublicKey;
+
+    constructor(label: string, secretKey: UserSecretKey) {
+        this.label = label;
+        this.secretKey = secretKey;
+        this.publicKey = secretKey.generatePublicKey();
+    }
+
+    static fromFile(path: PathLike, index: number = 0): UserPem {
+        return this.fromFileAll(path)[index];
+    }
+
+    static fromFileAll(path: PathLike): UserPem[] {
+        const resolvedPath = isAbsolute(path.toString())
+            ? resolve(path.toString())
+            : resolve(join(process.cwd(), path.toString()));
+        const text = readFileSync(resolvedPath, "utf-8");
+        return this.fromTextAll(text);
+    }
+
+    static fromText(text: string, index: number = 0): UserPem {
+        const items = this.fromTextAll(text);
+        return items[index];
+    }
+
+    static fromTextAll(text: string): UserPem[] {
+        const entries = PemEntry.fromTextAll(text);
+        const resultItems: UserPem[] = [];
+
+        for (const entry of entries) {
+            const secretKey = new UserSecretKey(entry.message.slice(0, USER_SEED_LENGTH));
+            const item = new UserPem(entry.label, secretKey);
+            resultItems.push(item);
+        }
+
+        return resultItems;
+    }
+
+    save(path: PathLike): void {
```

Comment:
Same pattern, same comments. Seems fine, can stay like this - can also be simplified (and leave more burden to the caller) :+1: :thought_balloon: